lib/parseOptions.js

```diff
@@ -10,6 +10,7 @@ import qs from 'querystring'
 
 export default function parseOptions (params) {
   const defaultOptions = {
+    environmentId: 'master',
```

Comment:
I am wondering if we should have this here or simply have `default: 'master'` in yargs configs. I did that for the import tool. the pros for that it will show in the help menu

Reply:
This default is for people using it as lib, not as CLI. yargs is CLI only

Reply:
ah yeah good point, forgot about that